test/uk/gov/hmrc/tai/connectors/EmploymentsConnectorSpec.scala

```diff
@@ -163,6 +163,95 @@ class EmploymentsConnectorSpec extends PlaySpec
 
   }
 
+  "EmploymentsConnector ceasedEmployments" must {
+
+    "call the employments API with a URL containing a service URL" when {
+      "the service URL is supplied" in {
+        val SUT = createSUT("test/service")
+        when(httpHandler.getFromApi(any())(any())).thenReturn(Future.successful(Json.parse(oneCeasedEmployment)))
+
+        val responseFuture = SUT.ceasedEmployments(nino, year)
+
+        Await.result(responseFuture, 5 seconds)
+        verify(httpHandler).getFromApi(Matchers.eq(s"test/service/tai/$nino/employments/year/${year.year}/status/ceased"))(any())
```

Comment:
I can see the other tests are doing the same thing in the this file but just for future reference, you are basically testing the mock here

Reply:
understood